src/pages/Filter/FilterPage.module.css

```diff
@@ -0,0 +1,22 @@
+.wrapper {
+  display: grid;
+  height: 100vh;
```

Comment:
Instead of adding `100vh` to each page, you could add this once in your App

Reply:
Will do this on a new branch